.github/workflows/ci.yaml

```diff
@@ -46,14 +47,15 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Setup dotnet 7.x
-        uses: actions/setup-dotnet@v1
+        uses: actions/setup-dotnet@v4
```

Comment:
This can be a single entry

```
      - 
        name: Setup .NET Core
        uses: actions/setup-dotnet@v4
        with:
          dotnet-version: |
            6.x
            7.x
            8.x
```